frontend/components/NewTooltipWrapper/NewTooltipWrapper.tsx

```diff
@@ -0,0 +1,94 @@
+import classnames from "classnames";
+import React from "react";
+import { Tooltip as ReactTooltip5, PlacesType } from "react-tooltip-5";
+
+import { uniqueId } from "lodash";
+
+interface INewTooltipWrapperBase {
+  children: string | JSX.Element;
+  position?: PlacesType;
+  isDelayed?: boolean;
+  underline?: boolean;
+  // Below two props used here to maintain the API of the old TooltipWrapper
+  // A clearer system would be to use the 3 below commented props, which describe exactly where they
+  // will apply, `element` being the element this tooltip will wrap. Associated logic is commented
+  // out, but ready to be used.
+  className?: string;
+  tooltipClass?: string;
+  // wrapperCustomClass?: string;
+  // elementCustomClass?: string;
+  // tipCustomClass?: string;
+  clickable?: boolean;
+}
+
+// Require either `tipContent` OR `customRender` prop
+interface INewTooltipWrapperCustomRender extends INewTooltipWrapperBase {
+  // see https://react-tooltip.com/docs/examples/render
+  customRender: (render: {
+    content: string | null;
+    activeAnchor: HTMLElement | null;
+  }) => any; // should actually return `ChildrenType` - TODO(jacob) - figure out how to type that
+  tipContent?: never;
+}
+
+export interface INewTooltipWrapperTipContent extends INewTooltipWrapperBase {
+  customRender?: never;
+  tipContent: string | JSX.Element;
+}
```

Comment:
Im not sure I understand these types and the use of type inheritance here. I can see it's to support some special tooltip rendering that react tooltip 5 can do but do we actually do this currently/need this for our UI? I can't see where we would need this for this component, and suggest we drop this and the type inheritance for a simpler `INewToolWrapperProps` interface.

Reply:
Sounds good

Reply:
The thinking was in case we ever want to use this component for more complex things, such as Wikipedia-like previews of CVE details (an idea that came up at the cx offsite – when you hover over a link on Wikipedia, you see a mini-page of that item without having to actually open it), having both interfaces would allow us to seamlessly use that custom render.

Do you think the flexibility for this kind of thing is worth keeping the dual interfaces?

Reply:
Simplified to single interface per discussion